### PR TITLE
Fix missing water tile in Infernal Views

### PR DESCRIPTION
**Pull request by HtotheTML**
Edited a stray tile in the lava river of Infernal Views to have its missing lava.

### CODE REVIEW

**Comment by duncanspumpkin** [comment]
Can you add a changelog entry. Also would be nice to attach before and after screenshots to this pull request.

**Comment by HtotheTML** [comment]
My apologies. here's what it looks like.
infernalbefore
infernalafter

**Comment by HtotheTML** [comment]
As far as adding the changelog entry, that's done with the labels, right? (first time doing a pull request)

**Comment by Broxzier** [comment]
The changelog is located in the `distribution` folder, called `changelog.txt`. Just add one entry for this fix. Keep the order in mind. Since there is no issue for this, you can use this PR's id (19641).

Since you're a first-time contributor, you may also add yourself to the `constibutors.md` file under Bug fixes.

File: distribution/changelog.txt [hunk 1]
```diff
@@ -1680,4 +1681,4 @@ This is the first fully implemented version of OpenRCT2. RCT2.EXE is no longer r
 - Feature: RollerCoaster Tycoon 1 scenarios can now be opened in the scenario editor or by using the ‘edit’ command line action.
 - Feature: The ‘have fun’ objective can now be selected in the scenario editor.
 - Feature: Twitch integration.
-- Fix: Litter bins now get full and require emptying by handymen.
\ No newline at end of file
+- Fix: Litter bins now get full and require emptying by handymen.
```
Comment (Broxzier) [inline distribution/changelog.txt:1684]: Please revert this accidental change. Your editor likely automatically added the EOL character at the end.

**Review by Broxzier — CHANGES_REQUESTED** [review]

File: distribution/changelog.txt (hunk 1 above)
Comment (HtotheTML) [reply distribution/changelog.txt:1684]: That's strange...I'm not seeing anything on my end in the editor.

Comment (HtotheTML) [reply distribution/changelog.txt]: Ok, I think it's gone now.

**Comment by Broxzier** [comment]
Please add yourself to the contributors.md file under Bug fixes, then this PR is good to go.

**Review by duncanspumpkin — APPROVED** [review]

**Review by Broxzier — APPROVED** [review]

**Pull request merged**